packages/integration-tests/utils/helpers.ts

```diff
@@ -46,6 +46,17 @@ async function getSentryTransactionRequest(page: Page, url?: string): Promise<Ev
   return (await getMultipleSentryTransactionRequests(page, 1, url))[0];
 }
 
+/**
+ * Get current Sentry session at the given URL, or the current page
+ *
+ * @param {Page} page
+ * @param {string} [url]
+ * @return {*}  {Promise<SessionContext>}
+ */
+async function getCurrentSession(page: Page, url?: string): Promise<SessionContext> {
+  return (await getMultipleSentryTransactionRequests(page, 1, url))[0];
```

Comment:
Technically this is not guaranteed to be a session right?

I think we should rename `getMultipleSentryTransactionRequests` -> `getMultipleSentryEnvelopeRequests` and directly call that in the tests. The tests can just handle picking the right request and asserting on it (they can even assert on the total amount of requests sent if needed).

Maybe we use `getMultipleSentryEnvelopeRequests` just for this PR, and then migrate the other test files in another PR.

Reply:
Yes, and https://github.com/getsentry/sentry-javascript/pull/4527 seems to solve the confusion. I'll update the PR with proper types after that gets merged, unless this one is urgent?

Reply:
Nothing using those types internally yet, I would just make the changes here and we can come back and update this after.